From 046c44fbff084918402947c505f7e814e019f732 Mon Sep 17 00:00:00 2001
From: Pardha Saradhi K <pardha.saradhi.kesapragada@intel.com>
Date: Fri, 5 Aug 2016 15:19:15 +0530
Subject: [PATCH 15/31] ASoC: Intel: Skylake: Support upto 8 Channels on
 Playback & Capture

Add capabilities to FE streams such that Playback and Capture
for all sample rates from 8K to 192K and all channels from
Mono to 8 Channel are supported

Change-Id: I318bf237aa3e9e6a841c6deeb3e8cd5bf300d997
Signed-off-by: Pardha Saradhi K <pardha.saradhi.kesapragada@intel.com>
Reviewed-on: https://git-gar-1.devtools.intel.com/gerrit/10792
Reviewed-by: B, Jayachandran <jayachandran.b@intel.com>
Reviewed-by: Babu, Ramesh <ramesh.babu@intel.com>
Tested-by: Jayanti, Satya Charitardha <satya.charitardha.jayanti@intel.com>

Conflicts:
	sound/soc/intel/skylake/skl-pcm.c

Conflicts:
	sound/soc/intel/skylake/skl-pcm.c
---
 sound/soc/intel/skylake/skl-pcm.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/sound/soc/intel/skylake/skl-pcm.c b/sound/soc/intel/skylake/skl-pcm.c
index 6c7babe..eadcbf6 100644
--- a/sound/soc/intel/skylake/skl-pcm.c
+++ b/sound/soc/intel/skylake/skl-pcm.c
@@ -1305,9 +1305,8 @@ struct skl_dsp_notify_ops cb_ops = {
 	.playback = {
 		.stream_name = "System Playback 2",
 		.channels_min = HDA_MONO,
-		.channels_max = HDA_STEREO,
-		.rates = SNDRV_PCM_RATE_48000 | SNDRV_PCM_RATE_16000 |
-			SNDRV_PCM_RATE_8000,
+		.channels_max = HDA_8_CH,
+		.rates = SNDRV_PCM_RATE_8000_192000 | SNDRV_PCM_RATE_KNOT,
 		.formats = SNDRV_PCM_FMTBIT_S16_LE | SNDRV_PCM_FMTBIT_S24_LE |
 			SNDRV_PCM_FMTBIT_S24_3LE | SNDRV_PCM_FMTBIT_S32_LE |
 			SNDRV_PCM_FMTBIT_FLOAT_LE,
-- 
1.9.1

